test/controllers/concerns/hal_actions_test.rb

```diff
@@ -1,29 +1,122 @@
 describe HalActions do
-  class HalActionsTestController < ActionController::Base
+
+  class Foo
+    include ActiveModel::Model
+
+    attr_accessor :id, :is_root_resource, :updated_at
+    cattr_accessor :_id
+
+    self._id = 1
+
+    def self.find(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `find(*)` if you want the method to accept any arguments but don't care about them.